Microsoft.ML.sln

```diff
@@ -133,6 +133,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.Samples", "doc
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.SamplesUtils", "src\Microsoft.ML.SamplesUtils\Microsoft.ML.SamplesUtils.csproj", "{11A5210E-2EA7-42F1-80DB-827762E9C781}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ML.Recommend", "src\Microsoft.ML.Recommend\Microsoft.ML.Recommend.csproj", "{C8E1772B-DFD9-4A4D-830D-6AAB1C668BB3}"
```

Comment:
>FAE04EC0 [](start = 10, length = 8)

I would have this project type match the 9A19... project id.

Reply:
I am not following. My change is not a part of tests. Could you add more context?

Reply:
change the GUID from FAE04EC0-301F-11D3-BF4B-00C04F79EFB to FAE04EC0-301F-11D3-BF4B-00C04F79EFB

---
In reply to: [225609269](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225609269) [](ancestors = 225609269)

Reply:
Fixed.

---
In reply to: [225351981](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225351981) [](ancestors = 225351981)